platformio.ini

```diff
@@ -11,24 +11,26 @@
 [env]
 monitor_speed = 115200 ; must remain at 115200 for improv
 board_build.filesystem = littlefs
+upload_port = /dev/ttyUSB0
```

Comment:
Can we remove this change?  It breaks the auto port detection

Reply:
Hah... yes, we both caught that.  Will remove.

Reply:
fixed